common/app/views/fragments/amp/stylesheets/liveBlog.scala.html

```diff
@@ -139,3 +139,71 @@
     text-align: center;
     padding-bottom: 1.5rem;
 }
+
+/* Toast
+========================================= */
+
+.toast__space-reserver {
```

Comment:
toast being the pop up thing when updates come in and you're already on the page https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/common/modules/ui/autoupdate.js#L57